src/libs/OptionsListUtils.js

```diff
@@ -351,7 +413,7 @@ function isCurrentUser(userDetails) {
  *
  * @param {Object} reports
  * @param {Object} personalDetails
- * @param {Number} activeReportID
+ * @param {String} activeReportID
```

Comment:
When did reportID become strings and are all reportID now strings in JS?

Reply:
This is being discussed in Slack to make sure we're getting to the right conclusion: https://expensify.slack.com/archives/C02HWMSMZEC/p1662382397049269

Regardless, we're not treating them consistently everywhere yet, but we want to!